crates/ruma-events/src/room/message/audio.rs

```diff
@@ -92,6 +95,14 @@ impl AudioMessageEventContent {
         Self { info: info.into(), ..self }
     }
 
+    /// Computes the filename for the audio file as defined by the [spec](https://spec.matrix.org/v1.11/client-server-api/#maudio).
```

Comment:
I would have actually linked to the media captions section, it's easier to read than the event's definition, imo. Also we always link to the `latest` spec:

```suggestion
    /// Computes the filename for the audio file as defined by the [spec](https://spec.matrix.org/latest/client-server-api/#media-captions).
```

Same with the other links below.

Reply:
Ok, makes sense, updated 👍